ietf-optical-impairment-topology.tree

```diff
@@ -182,13 +182,13 @@ module: ietf-optical-impairment-topology
                 |           |  +--ro lower-frequency    frequency-thz
                 |           |  +--ro upper-frequency    frequency-thz
                 |           +--ro actual-gain
-                |           |       l0-types:ratio-in-db-or-null
+                |           |       l0-types:gain-in-db-or-null
                 |           +--ro tilt-target
                 |           |       l0-types:decimal-2-digits-or-null
```

Comment:
tilt target isusually a value in dB I suggest to add the unit in the yang description, and/or to change the type into ratio-in-db-or-null: because it can be positive or negative

Reply:
We have removed the ratio-in-db-or-null data type from l0-types: maybe better to keep it as a decimal-2-digits-or-null type and add units "dB"

Reply:
Addressed in https://github.com/ietf-ccamp-wg/draft-ietf-ccamp-optical-impairment-topology-yang/pull/135/commits/62ec3365bb97f7d5b1a2b3135a70091c2f52cbc5

Co-authored-by: sergio belotti <sergio.belotti@nokia.com>